### PR TITLE
fix typos in comments/logs

### PR DESCRIPTION
**Pull request by AndrewMeadows**
This PR has minor fixes for typos in a comment and some logs.

### CODE REVIEW

File: src/classes/HiFiAxisConfiguration.ts [hunk 1]
```diff
@@ -108,58 +108,58 @@ export class HiFiAxisUtilities {
 
         // START into-screen/out-of-screen axis error checking
         if (axisConfiguration.intoScreenAxis === HiFiAxes.PositiveX && axisConfiguration.outOfScreenAxis !== HiFiAxes.NegativeX) {
-            HiFiLogger.error(`Invalid axis configuration!\nRight Axis is ${axisConfiguration.intoScreenAxis}, and Left Axis is ${axisConfiguration.outOfScreenAxis}!`);
+            HiFiLogger.error(`Invalid axis configuration!\nIntoScreen Axis is ${axisConfiguration.intoScreenAxis}, and OutOfScree Axis is ${axisConfiguration.outOfScreenAxis}!`);
```
Comment (hd23408) [inline src/classes/HiFiAxisConfiguration.ts:111]: Missing 'n' in OutOfScreen

File: src/classes/HiFiAxisConfiguration.ts [hunk 2]
```diff
@@ -108,58 +108,58 @@ export class HiFiAxisUtilities {
...
         }
         if (axisConfiguration.outOfScreenAxis === HiFiAxes.PositiveX && axisConfiguration.intoScreenAxis !== HiFiAxes.NegativeX) {
-            HiFiLogger.error(`Invalid axis configuration!\nRight Axis is ${axisConfiguration.intoScreenAxis}, and Left Axis is ${axisConfiguration.outOfScreenAxis}!`);
+            HiFiLogger.error(`Invalid axis configuration!\nIntoScreen Axis is ${axisConfiguration.intoScreenAxis}, and OutOfScree Axis is ${axisConfiguration.outOfScreenAxis}!`);
             isValid = false;
         }
 
         if (axisConfiguration.intoScreenAxis === HiFiAxes.PositiveY && axisConfiguration.outOfScreenAxis !== HiFiAxes.NegativeY) {
-            HiFiLogger.error(`Invalid axis configuration!\nRight Axis is ${axisConfiguration.intoScreenAxis}, and Left Axis is ${axisConfiguration.outOfScreenAxis}!`);
+            HiFiLogger.error(`Invalid axis configuration!\nIntoScreen Axis is ${axisConfiguration.intoScreenAxis}, and OutOfScree Axis is ${axisConfiguration.outOfScreenAxis}!`);
```
Comment (hd23408) [inline src/classes/HiFiAxisConfiguration.ts:120]: 'n' again

File: src/classes/HiFiAxisConfiguration.ts [hunk 3]
```diff
@@ -108,58 +108,58 @@ export class HiFiAxisUtilities {
...
             isValid = false;
         }
         if (axisConfiguration.outOfScreenAxis === HiFiAxes.PositiveY && axisConfiguration.intoScreenAxis !== HiFiAxes.NegativeY) {
-            HiFiLogger.error(`Invalid axis configuration!\nRight Axis is ${axisConfiguration.intoScreenAxis}, and Left Axis is ${axisConfiguration.outOfScreenAxis}!`);
+            HiFiLogger.error(`Invalid axis configuration!\nIntoScreen Axis is ${axisConfiguration.intoScreenAxis}, and OutOfScree Axis is ${axisConfiguration.outOfScreenAxis}!`);
```
Comment (hd23408) [inline src/classes/HiFiAxisConfiguration.ts:124]: (you get the idea)

**Review by hd23408 — CHANGES_REQUESTED** [review]

File: src/classes/HiFiAxisConfiguration.ts [hunk 4]
```diff
@@ -108,58 +108,58 @@ export class HiFiAxisUtilities {
 
         // START into-screen/out-of-screen axis error checking
         if (axisConfiguration.intoScreenAxis === HiFiAxes.PositiveX && axisConfiguration.outOfScreenAxis !== HiFiAxes.NegativeX) {
-            HiFiLogger.error(`Invalid axis configuration!\nRight Axis is ${axisConfiguration.intoScreenAxis}, and Left Axis is ${axisConfiguration.outOfScreenAxis}!`);
+            HiFiLogger.error(`Invalid axis configuration!\nIntoScreen Axis is ${axisConfiguration.intoScreenAxis}, and OutOfScree Axis is ${axisConfiguration.outOfScreenAxis}!`);
             isValid = false;
         }
         if (axisConfiguration.outOfScreenAxis === HiFiAxes.PositiveX && axisConfiguration.intoScreenAxis !== HiFiAxes.NegativeX) {
-            HiFiLogger.error(`Invalid axis configuration!\nRight Axis is ${axisConfiguration.intoScreenAxis}, and Left Axis is ${axisConfiguration.outOfScreenAxis}!`);
+            HiFiLogger.error(`Invalid axis configuration!\nIntoScreen Axis is ${axisConfiguration.intoScreenAxis}, and OutOfScree Axis is ${axisConfiguration.outOfScreenAxis}!`);
```
Comment (zfox23) [inline src/classes/HiFiAxisConfiguration.ts:115]:
```suggestion
            HiFiLogger.error(`Invalid axis configuration!\nIntoScreen Axis is ${axisConfiguration.intoScreenAxis}, and OutOfScreen Axis is ${axisConfiguration.outOfScreenAxis}!`);
```

File: src/classes/HiFiAxisConfiguration.ts (hunk 1 above)
Comment (zfox23) [reply src/classes/HiFiAxisConfiguration.ts:111]:
```suggestion
            HiFiLogger.error(`Invalid axis configuration!\nIntoScreen Axis is ${axisConfiguration.intoScreenAxis}, and OutOfScreen Axis is ${axisConfiguration.outOfScreenAxis}!`);
```

File: src/classes/HiFiAxisConfiguration.ts (hunk 2 above)
Comment (zfox23) [reply src/classes/HiFiAxisConfiguration.ts:120]:
```suggestion
            HiFiLogger.error(`Invalid axis configuration!\nIntoScreen Axis is ${axisConfiguration.intoScreenAxis}, and OutOfScreen Axis is ${axisConfiguration.outOfScreenAxis}!`);
```

File: src/classes/HiFiAxisConfiguration.ts (hunk 3 above)
Comment (zfox23) [inline src/classes/HiFiAxisConfiguration.ts:124]:
```suggestion
            HiFiLogger.error(`Invalid axis configuration!\nIntoScreen Axis is ${axisConfiguration.intoScreenAxis}, and OutOfScreen Axis is ${axisConfiguration.outOfScreenAxis}!`);
```

File: src/classes/HiFiAxisConfiguration.ts [hunk 5]
```diff
@@ -108,58 +108,58 @@ export class HiFiAxisUtilities {
...
         }
         if (axisConfiguration.outOfScreenAxis === HiFiAxes.PositiveY && axisConfiguration.intoScreenAxis !== HiFiAxes.NegativeY) {
-            HiFiLogger.error(`Invalid axis configuration!\nRight Axis is ${axisConfiguration.intoScreenAxis}, and Left Axis is ${axisConfiguration.outOfScreenAxis}!`);
+            HiFiLogger.error(`Invalid axis configuration!\nIntoScreen Axis is ${axisConfiguration.intoScreenAxis}, and OutOfScree Axis is ${axisConfiguration.outOfScreenAxis}!`);
             isValid = false;
         }
 
         if (axisConfiguration.intoScreenAxis === HiFiAxes.PositiveZ && axisConfiguration.outOfScreenAxis !== HiFiAxes.NegativeZ) {
-            HiFiLogger.error(`Invalid axis configuration!\nRight Axis is ${axisConfiguration.intoScreenAxis}, and Left Axis is ${axisConfiguration.outOfScreenAxis}!`);
+            HiFiLogger.error(`Invalid axis configuration!\nIntoScreen Axis is ${axisConfiguration.intoScreenAxis}, and OutOfScree Axis is ${axisConfiguration.outOfScreenAxis}!`);
```
Comment (zfox23) [inline src/classes/HiFiAxisConfiguration.ts:129]:
```suggestion
            HiFiLogger.error(`Invalid axis configuration!\nIntoScreen Axis is ${axisConfiguration.intoScreenAxis}, and OutOfScreen Axis is ${axisConfiguration.outOfScreenAxis}!`);
```

File: src/classes/HiFiAxisConfiguration.ts [hunk 6]
```diff
@@ -108,58 +108,58 @@ export class HiFiAxisUtilities {
...
             isValid = false;
         }
         if (axisConfiguration.outOfScreenAxis === HiFiAxes.PositiveZ && axisConfiguration.intoScreenAxis !== HiFiAxes.NegativeZ) {
-            HiFiLogger.error(`Invalid axis configuration!\nRight Axis is ${axisConfiguration.intoScreenAxis}, and Left Axis is ${axisConfiguration.outOfScreenAxis}!`);
+            HiFiLogger.error(`Invalid axis configuration!\nIntoScreen Axis is ${axisConfiguration.intoScreenAxis}, and OutOfScree Axis is ${axisConfiguration.outOfScreenAxis}!`);
```
Comment (zfox23) [inline src/classes/HiFiAxisConfiguration.ts:133]:
```suggestion
            HiFiLogger.error(`Invalid axis configuration!\nIntoScreen Axis is ${axisConfiguration.intoScreenAxis}, and OutOfScreen Axis is ${axisConfiguration.outOfScreenAxis}!`);
```

**Review by zfox23 — CHANGES_REQUESTED** [review]

**Review by zfox23 — APPROVED** [review]

**Pull request merged**